kge/job/search.py

```diff
@@ -11,7 +11,7 @@ class SearchJob(Job):
     Provides functionality for scheduling training jobs across workers.
     """
 
-    def __init__(self, config, dataset, parent_job=None):
+    def __init__(self, config, dataset, parent_job=None, init=True):
```

Comment:
Why the init=True here (and elsewhere)?

Reply:
this made it possible to not create a model on init of training job. Added everywhere else to keep signature persistent.
Now replaced with model parameter to directly set the model. Only added to training job signature